Source/VCGeneration/VCGen.cs

```diff
@@ -911,7 +929,7 @@ void SplitAndVerify(Implementation impl, Dictionary<TransferCmd, ReturnCmd> goto
               {
                 if (prover_failed)
```

Comment:
This is another annoying divergence from the convention of using camelCase for variable names.  Since you are modifying this file anyway, could you please take a shot at renaming names like "prover_failed" to "proverFailed" (and others).  But I won't insist on this change for this PR.  Perhaps you would consider doing this change in a separate PR.

Reply:
Let's merge this in and I will soon create another PR for variable renaming.